src/interp/interp.cc

```diff
@@ -203,6 +203,7 @@ Store::Store(const Features& features) : features_(features) {
   Ref ref{objects_.New(new Object(ObjectKind::Null))};
   assert(ref == Ref::Null);
   roots_.New(ref);
+  gc_context_.call_depth = 0;
```

Comment:
This should be the definition of GCContext I think.  (can you just do `int call_depth = 0;` there?)